modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -49,30 +50,32 @@ namespace {{packageName}}.Client
 
         /// <summary>
         /// Initializes a new instance of the <see cref="ApiClient" /> class
-        /// with default configuration and base path ({{{basePath}}}).
+        /// with default configuration and base path (<see cref="SERVER_DEFAULT_ADDRESS"/>).
         /// </summary>
-        public ApiClient()
-        {
+        public ApiClient() {
```

Comment:
@iamkarlson `{` should start with a new line so as to conform to C# style guide.